main.tf

```diff
@@ -171,6 +171,11 @@ locals {
       memory_request = instance.memory_request
       memory_limit   = instance.memory_limit
 
+
+      balancer_cpu_request    = lookup(instance, "balancer_cpu_request", null)
+      balancer_memory_request = lookup(instance, "balancer_memory_cpu_request", null)
+      balancer_memory_limit   = lookup(instance, "balancer_memory_cpu_limit", null)
+
```

Comment:
Apologies ... usually I don't like to muddy PRs ...
But, the environmentd_version isn't included here.  Could we patch to include that?

Reply:
Actually, ignore.  I'll have to make changes to the examples/simple anyways ... so, will make a separate PR to take care of that.